readme.md

```diff
@@ -55,6 +55,35 @@ By convention, when the artifact is a library, the files are called
   or regression) and turns it into a binary classification problem
   with the two classes "normal" and "anomalous".
 
+## How to install WhizzML packages
```

Comment:
i would write some kind of introduction here saying that there are three kinds of installable whizzml artifacts in this repo, identified by the field "kind" in their metadata: libraries, scripts and packages.  The latter are compounds of libraries and scripts, possibly with interdependencies, meant to be installed together.